### DISCUSSION TITLE
only slight changes in prediction

### DISCUSSION THREAD

**Discussion by FlorianWieser1** [wk989898/ARES-implement#3] [Q&A] [answered]
Hi wk989898,

Thanks a lot for you fix! Still I wonder how you were able to train the model for meaningful predictions. Whatever I try, with your or with the original ARES implementation, I always end up with predictions that change only slightly (in the 3rd digit).
 
out:5.858390808105469 rms:7.942 gap:2.083609104156494
out:5.858109951019287 rms:12.412 gap:6.553889751434326
out:5.859871864318848 rms:13.533 gap:7.673128128051758
out:5.857667446136475 rms:10.421 gap:4.5633320808410645
...
Here I tried the same data and hyperparameters as you did.
Any ideas?
Thank you very much!

**Comment by wk989898** [comment, marked as answer]
https://purl.stanford.edu/bn398fc4306 File4
